### PR TITLE
Add portuguese translation to deploy_to_githubpages and element_ref

### PR DESCRIPTION
**Pull request by thaymara**

### CODE REVIEW

File: deploy_to_githubpages.md [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
+* Você precisa ter um usuário no Github
+* Você precisa criar um repositório para o seu projeto.
+* Você precisa fazer um commit de todas as suas alterações que você fez no projeto
+* Você precisa instalar o angular-cli-ghpages
 
-## Creating a Github user
-If you already have a Github user you can skip this step.
-To Create a Github user go to Github: https://github.com/
-Fill the regetration form and make sure to validate your email address.
+## Criando um usuáario no Github
```
Comment (cyz) [inline deploy_to_githubpages.md]: Hi, @thaymara, please, can you review the word "usuário"? Thanks

File: deploy_to_githubpages.md [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+Se você já possui um usuário no Github você pode pular esse passo.
+Criar um usuário no Github: https://github.com/
+Preencha o formulário de registro e faça a validação de seu e-mail
 
-## Create your App repository
-After logging in to Github.
-Click on the `Start a project` button, and name the repository `ng-girls-todo` or any other name you like.
+## Criar um reposiório de seu APP
```
Comment (cyz) [inline deploy_to_githubpages.md]: Hi, @thaymara, please, can you review the word "repositório"? Thanks

**Comment by thaymara** [comment]
Hi @cyz I sent the fixes :)
If there is more fixes please let me know.

**Comment by hi-hi-ray** [comment]
@cyz Eu acredito que ela já tenha mudado o que você solicitou. =]

**Pull request merged**